modules/tfm/zephyr/CMakeLists.txt

```diff
@@ -20,3 +20,12 @@ endif()
 set_property(GLOBAL PROPERTY
   tfm_PM_HEX_FILE $<TARGET_PROPERTY:tfm,TFM_S_HEX_FILE>
   )
+
+if (CONFIG_TFM_MINIMAL)
+  set_property(TARGET zephyr_property_target
+    APPEND PROPERTY TFM_CMAKE_OPTIONS
+    -DPLATFORM_DUMMY_NV_COUNTERS=FALSE
+    -DPS_TEST_NV_COUNTERS=OFF
+    -DPLATFORM_NV_COUNTER_MODULE_DISABLED=TRUE
```

Comment:
are those not having Zephyr TF-M Kconfig equivalents ?

Reply:
No, we should consider adding this, but I'm starting to see a pattern which we might want to address (a lot of code for every option we want to set in TF-M)

Reply:
we might actually need a kconfig for this since we have to check if its enabled to decide whether or not the tfm-extra partition should exist.